scripts/setup-emlinux

```diff
@@ -50,11 +50,10 @@ if [ -z "$TEMPLATECONF" ] && [ "${EML_BUILDDIR_SETUP_DONE}" = "false" ]; then
     # Add config
     echo "DISTRO = \"emlinux\"" >> conf/local.conf
     echo "PACKAGE_CLASSES = \"package_deb\"" >> conf/local.conf
-    echo "BBMASK = \"meta-debian/recipes-core\"" >> conf/local.conf
     # nsswitch.conf is not provided by debian base-file package but systemd requires it.
     # use poky's base-files package temporary.
-    echo "BBMASK .= \"|meta-debian/recipes-debian/base-files/base-files_debian.bb\"" >> conf/local.conf
     echo "DL_DIR = \"\${TOPDIR}/../downloads\"" >> conf/local.conf
+    cat ${REPOS}/meta-emlinux/scripts/bbmask.conf >> conf/local.conf
```

Comment:
This commit should create the bbmask.conf, which includes mask-configs about the recipes-core and the base-files.
In your next commit, please add the rest configs.

Reply:
Is this comment as a commit message for setup-emlinux?